book/src/account-authority.md

```diff
@@ -0,0 +1,23 @@
+# Account Authority #
+
+## Problem ##
+
+There's no extant general facility to re-key or to delegate authority of an account in Solana. This manifests as a problem in several ways:
```

Comment:
Can you expand on what you mean by `authority` precisely and how it differs from ownership. Is it the same as what ERC-20 calls a `delegate`?

Reply:
looks similar.  we have choices here.  looks like 'delegate' is someone who can withdraw tokens, but I'm using 'authority' because I want to distinguish authorized instructions from other things (like transfers).